src/app/modules/card/card.component.scss

```diff
@@ -0,0 +1,12 @@
+.ukm-card{
+  width: 10vw;
+  height: 5vh;
```

Comment:
don't use vw/vh, it'll make the size becomes not-proportional with other components (e.g. fonts, components inside the card, etc.)